PocketExample/Pocket/Models/Transaction.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Transaction.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 4/24/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+
+class Transaction {
+    
+    var nonce: String
+    var recipient: String
+    var value: String
+    var data: NSDictionary
+    var signed: String
+    
+    init(nonce: String, recipient: String, value: String, data: NSDictionary) {
```

Comment:
Add the initial default values to the property declarations, like `self.signed = ""`. 

You can remove the initializer if you make this a struct due to memberwise initialization.